### PR TITLE
STREAMS-1279 - Describe webhook exchanges API.

### PR DESCRIPTION
**Pull request by rbissey-axway**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 1]
```diff
@@ -76,6 +76,66 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Getting webhook exchanges for a subscription
+
+In order to know all exchanges done with the `webhookUrl` of a subscription during the last 5 minutes, simply do the following GET request:
```
Comment (guillaumelamirand) [inline content/en/docs/subscribers/subscriber-webhook.md:81]: I will say 'all exchanges done for a subscription'

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 2]
```diff
@@ -76,6 +76,66 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges`
+
+The default time window of the retrieved history can be changed with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z).
```
Comment (guillaumelamirand) [inline content/en/docs/subscribers/subscriber-webhook.md:85]: Provide an full example as above for default value

File: content/en/docs/subscribers/subscriber-webhook.md (hunk 1 above)
Comment (rbissey-axway) [reply content/en/docs/subscribers/subscriber-webhook.md]: Updated!

File: content/en/docs/subscribers/subscriber-webhook.md (hunk 2 above)
Comment (rbissey-axway) [reply content/en/docs/subscribers/subscriber-webhook.md]: A second example with full parameters has been added.

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 3]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Getting webhook exchanges for a subscription
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:79]:
```suggestion
## Getting the webhook notification history for a subscription
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 4]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+In order to know all exchanges done for a subscription during the last 5 minutes, simply do the following GET request:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:81]:
```suggestion
In order to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes, just do the following `GET` request:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 5]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges`
+
+The default time window of the retrieved history can be changed with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z):
+
+`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges?start=2021-01-10T10:13:30Z&end=2021-01-10T10:13:32Z`
+
+### Webhook exchanges status codes
+
+Below the list of HTTP status codes that can be returned when trying to get exchanges history:
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:91]:
```suggestion
Below the list of HTTP status codes that can be returned when interacting with the exchanges history:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 6]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+| Code | Comment |
+|------|---------|
+| 200 Ok | Indicates that the exchanges history requested is valid and has been retrieved. |
+| 400 Bad Request | Indicates that the provided data are invalid. |
```
Comment (allandenis38) [inline content/en/docs/subscribers/subscriber-webhook.md:96]:
```suggestion
| 400 Bad Request | Indicates that the request is invalid. |
```

**Review by allandenis38 — CHANGES_REQUESTED** [review]

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@lbadenhop This PR is ready for your review.
Preview: https://deploy-preview-56--streams-open-docs.netlify.app/

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 7]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
 | 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
 | 404 Not found | Indicates that the requested URL or subscription requested does not exist. |
 
+## Getting the webhook notification history for a subscription
+
+In order to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes, just do the following `GET` request:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:81]:
```suggestion
Use the following `GET` request to retrieve the history of webhook exchanges (requests and responses) that have occurred for a subscription in the last 5 minutes:
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 8]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges`
+
+The default time window of the retrieved history can be changed with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z):
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:85]:
```suggestion
You can change the default time window of the retrieved history with `start` and `end` query params (date-time in ISO 8601 format, eg: 2021-01-10T10:13:32Z):
```

File: content/en/docs/subscribers/subscriber-webhook.md [hunk 9]
```diff
@@ -76,6 +76,68 @@ Below the list of HTTP status codes that can be returned when trying to get a ka
...
+
+`GET /subscribers/webhook/subscriptions/{subscriptionId}/exchanges?start=2021-01-10T10:13:30Z&end=2021-01-10T10:13:32Z`
+
+### Webhook exchanges status codes
+
+Below the list of HTTP status codes that can be returned when interacting with the exchanges history:
```
Comment (lbadenhop) [inline content/en/docs/subscribers/subscriber-webhook.md:91]:
```suggestion
The following HTTP status codes can be returned when interacting with the exchanges history:
```

**Review by lbadenhop — COMMENTED** [review]
See suggested changes.

**Pull request merged**